_inc/client/plans/plan-grid.jsx

```diff
@@ -114,6 +114,7 @@ class PlanGrid extends React.Component {
 				<ButtonGroup>
 					{ map( periods, ( periodLabel, periodName ) => (
 						<Button
+							key={ periodLabel }
```

Comment:
Can we use the `periodName` instead of the `periodLabel` here? Also have some sort of a prefix/suffix. Example:
```suggestion
							key={ 'plan-period-button-' + periodName }
```